debian/nnstreamer-nntrainer-dev.install

```diff
@@ -0,0 +1 @@
+/usr/lib/*/libnnstreamer_filter_nntrainer.a
```

Comment:
Are we using static?

Reply:
its released in tizen, so debian matches it for now.
if it is to be removed, then then need to check for tizen first.

Reply:
I see, thanks